src/Utils/ConnectionComponent.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react';
+import LoadingAnimation from './../Utils/LoadingAnimation';
+import ErrorMessage from './../Utils/ErrorMessage';
+import Config from './../config/config.js';
+import BackendMock from './MockData/BackendMock.js';
+
+class ConnectionComponent extends Component {
+
+	render(promiseState) {
+		if (promiseState.pending) {
+			return <LoadingAnimation/>;
+		} else if (promiseState.rejected) {
+			return <ErrorMessage message={promiseState.reason.message}/>;
+		}
+		return null;
+	}
+	
+	static switchFetch(input, init) {
+		if(Config.useBackendMock) {
+			const promise = new Promise(resolve => resolve(BackendMock.handleRequest(input, init)));
+			console.log(promise);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)